### PR TITLE
AISDK-169: Add transcriber and start ts

### PR DESCRIPTION
**Pull request by Chanc3**

### CODE REVIEW

File: src/main/java/ai/rev/speechtotext/StreamingClient.java [hunk 1]
```diff
@@ -202,6 +202,14 @@ private String buildURL(
         urlBuilder.addQueryParameter(
             "delete_after_seconds", String.valueOf(sessionConfig.getDeleteAfterSeconds()));
       }
+      if (sessionConfig.getStartTs() != null) {
+        urlBuilder.addQueryParameter(
+                "start_ts", String.valueOf(sessionConfig.getStartTs()));
+      }
```
Comment (k-weng) [inline src/main/java/ai/rev/speechtotext/StreamingClient.java]: are we using 2-space indents? If so
```suggestion
        urlBuilder.addQueryParameter(
          "start_ts", String.valueOf(sessionConfig.getStartTs()));
      }
```

File: src/main/java/ai/rev/speechtotext/StreamingClient.java [hunk 2]
```diff
@@ -202,6 +202,14 @@ private String buildURL(
...
+      if (sessionConfig.getTranscriber() != null) {
+        urlBuilder.addQueryParameter(
+                "transcriber", sessionConfig.getTranscriber());
+      }
```
Comment (k-weng) [inline src/main/java/ai/rev/speechtotext/StreamingClient.java]:
```suggestion
        urlBuilder.addQueryParameter(
          "transcriber", sessionConfig.getTranscriber());
      }
```

File: src/test/java/ai/rev/speechtotext/unit/RevAiStreamingClientOptionalParametersTest.java [hunk 3]
```diff
@@ -139,6 +154,14 @@ private void assertStreamOptions(RecordedRequest request) throws UnsupportedEnco
       assertThat(request.getPath())
           .contains("delete_after_seconds=" + sessionConfig.getDeleteAfterSeconds());
     }
+    if (sessionConfig.getStartTs() != null) {
+      assertThat(request.getPath())
+              .contains("start_ts=" + sessionConfig.getStartTs());
```
Comment (k-weng) [inline src/test/java/ai/rev/speechtotext/unit/RevAiStreamingClientOptionalParametersTest.java]: Don't think this needs two lines
```suggestion
      assertThat(request.getPath()).contains("start_ts=" + sessionConfig.getStartTs());
```
But if two lines is wanted, will need proper indentation
```suggestion
      assertThat(request.getPath())
        .contains("start_ts=" + sessionConfig.getStartTs());
```

File: src/test/java/ai/rev/speechtotext/unit/RevAiStreamingClientOptionalParametersTest.java [hunk 4]
```diff
@@ -139,6 +154,14 @@ private void assertStreamOptions(RecordedRequest request) throws UnsupportedEnco
...
+    }
+    if (sessionConfig.getTranscriber() != null) {
+      assertThat(request.getPath())
+              .contains("transcriber=" + sessionConfig.getTranscriber());
+    }
```
Comment (k-weng) [inline src/test/java/ai/rev/speechtotext/unit/RevAiStreamingClientOptionalParametersTest.java]: Same comment as above

**Review by k-weng — CHANGES_REQUESTED** [review]
Few comments but overall looks good, will need to update package version (I think)

File: examples/StreamingFromLocalFile.java [hunk 5]
```diff
@@ -35,6 +35,8 @@ public static void main(String[] args) throws InterruptedException {
     sessionConfig.setFilterProfanity(true);
     sessionConfig.setRemoveDisfluencies(true);
     sessionConfig.setDeleteAfterSeconds(2592000); // 30 days in seconds
+    sessionConfig.setStartTs(0.0);
+    sessionConfig.setTranscriber("machine_v2");
```
Comment (kbridbur) [inline examples/StreamingFromLocalFile.java]: Same comment as on the node sdk. This should not be the default for the example

**Review by kbridbur — APPROVED** [review]

**Review by k-weng — APPROVED** [review]

**Pull request merged**